model/README.md

```diff
@@ -196,6 +199,76 @@ model in augmented reality to allow the user to visualize it at real scale in th
 surroundings. To support this, new DOM APIs may be added or the existing HTML Fullscreen API extended
 via more [FullscreenOptions](https://fullscreen.spec.whatwg.org/#dictdef-fullscreenoptions) properties.
 
+### Controlling the camera
+
+While by setting the `interactive` property the author will allow a built-in behavior such that dragging
+over a `<model>` element would result in modifying the camera, we propose to allow authors direct control
+of the camera via DOM APIs. An initial proposition would be to add an `HTMLModelElementCamera`:
+
+```idl
+dictionary HTMLModelElementCamera {
```

Comment:
But you can't change the camera location (affection foreshortening)? Calling this "the camera" might be a little shortsighted; it's camera coordinates.